server/@types/bapv.d.ts

```diff
@@ -156,9 +156,8 @@ export type VisitsReviewListItem = {
   visitDates: string[]
 }
 
-export interface Prison extends PrisonName {
-  policyNoticeDaysMin: number
-  maxTotalVisitors: number
+export interface Prison extends Omit<PrisonDto, 'code'> {
```

Comment:
Change `Prison` to be based on `PrisonDto` (as this has various config values that will be needed), **but** switch property name `code` to `prisonId` for consistency with existing code.